Products/zms/standard.py

```diff
@@ -340,9 +340,8 @@ def set_response_headers_cache(context, request=None, cache_max_age=24*3600, cac
       expire_datetime_gmt = expire_datetime.toZone('GMT')
 
       if t1 > t0 and cache_s_maxage > expire_in_secs:
-        request.RESPONSE.setHeader('Expires', expire_datetime_gmt.asdatetime().strftime('%a, %d %b %Y %H:%M:%S %Z'))
-        request.RESPONSE.setHeader('Cache-Control','s-maxage={}, max-age={}, public, must-revalidate, proxy-revalidate'.format(expire_in_secs, expire_in_secs))

```

Comment:
Check if Expires + X-Accel-Expires causes interferences
-> or if removal is intentional

Reply:
SHOULD BE OK